manager/apis/app/v1/interface_types.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 IBM Corp.
```

Comment:
I have deleted this file when cleaning apis.

Reply:
ok, probably during the rebase it slid into. Thanks